src/backbone.intercept.js

```diff
@@ -60,9 +60,9 @@ Backbone.Intercept = {
       return;
     }
 
-    // If the Ctrl key is held, we want to open in a new tab.
-    bypass = e.ctrlKey;
-    if (bypass) {
+    // If the platform’s modifier key is held, we want to open in a new tab.
+    var platform = navigator.platform.toLowerCase();
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/NavigatorID/platform It seems like such a basic part of any browser’s API, that any browser will always return a string. The string can be  manipulated, of course, but then the user would be to blame. I don’t think any browsers has been pretending to be any other browser for a long, long while? To me, even expanding the condition to look for an empty platform string seems unnecessary; http://stackoverflow.com/a/19883965/1209356